Assets/Modules/Enemies/EnemySpawnManager.cs

```diff
@@ -1,38 +1,78 @@
 using System.Collections;
 using System.Collections.Generic;
+using UnityEditor.Rendering;
 using UnityEngine;
 
 public class EnemySpawnManager : MonoBehaviour
 {
+    [Header("SHOOTER")]
     [SerializeField]
     private float timeSpawnShooter = 10f;
 
     [SerializeField]
-    private GameObject shooter;
+    private GameObject shooterPrefab;
 
-    private float timer;
+    public bool isSpawningShooter;
```

Comment:
Where are you assigning the value of this boolean? I have the same argument as "IsAlive" boolean in PlayerManager. This should be only readable for other classes and they shouldn't be able to change its value

Reply:
For now, this one, I'm assigining it myself in the inspector, don't have any other script touching this. I would perhaps, on game manager, or difficulty manager, to have the EnemySpawnManager stop or start instancing certain types of enemies depending on the level/difficulty later.

Reply:
oh then if you're using it in the inspector it's better to make it:

[Serializefield] private bool isSpawningShooter;

isAlive case was a bit different than this sorry for confusing you 😅